src/deploy/NVA_build/noobaa_init.sh

```diff
@@ -151,6 +163,9 @@ then
 elif [ "${RUN_INIT}" == "init_mongo" ]
 then
   init_pod
+elif [ "${RUN_INIT}" == "init_endpoint" ]
```

Comment:
Why not just `endpoint`? Seems like you copied that from `init_mongo` which is just an init container and in the endpoint case this is the actual endpoint run command.

Reply:
All the functions start with init, init_agent, init_noobaa_server, init_mongo. It describes the fact that the function's code is initializing the environment and then call run_<component>_container